plugins/lsp/conform-nvim.nix

```diff
@@ -0,0 +1,89 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.conform-nvim;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.conform-nvim =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "conform-nvim";
+
+      package = helpers.mkPackageOption "conform-nvim" pkgs.vimPlugins.conform-nvim;
+
+      # Map of filetype to formatters
+      formattersByFt =
+        helpers.defaultNullOpts.mkNullable types.attrs
+        "see documentation"
+        ''
+          Map of filetype to formatters
+          lua = { "stylua" },
+          Conform will run multiple formatters sequentially
+          python = { "isort", "black" },
+          Use a sub-list to run only the first available formatter
+          javascript = { { "prettierd", "prettier" } },
+          Use the "*" filetype to run formatters on all filetypes.
+          ["*"] = { "codespell" },
+          Use the "_" filetype to run formatters on filetypes that don't
+          have other formatters configured.
+          ["_"] = { "trim_whitespace" },
```

Comment:
could you put the code samples inside \` \` (the documentation is markdown. Could you also translate the code samples to their Nix form ?

Reply:
like this?
```nix
`''["*"]" = [ "codespell" ];`
```

Reply:
simply `"*" = [ "codespell" ]`, the `["..."] = ` is equalivalent to nix `"..." = `

Reply:
But please put it in `{ ...; }` for it to be valid Nix